app/models/issue_assigner.rb

```diff
@@ -9,46 +9,51 @@ def initialize(user, subscriptions)
 
   def assign!
     subscriptions.each do |sub|
-      Array.new(sub.email_limit) do
-        assign_issue_for_sub(sub)
-      end
+      assign_issues_for_sub(sub, email_limit: sub.email_limit)
     end
     self
   end
 
   private
 
-  def assign_issue_for_sub(sub)
-    issue = Issue.find_by_sql(<<-SQL
+  def assign_issues_for_sub(sub, email_limit:)
+    issues = Issue.find_by_sql([%Q{
                 SELECT
                   *
                 FROM
                   issues
                 WHERE
-                  repo_id = '#{sub.repo_id}' and
-                  state   = '#{Issue::OPEN}'
+                  repo_id = :repo_id AND
+                  state   = :repo_state
```

Comment:
@schneems I think we should rename this to :issue_state, instead of :repo_state.